api/database/migrations/20200127000435-modify-role-superadmin-volunteer.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.changeColumn("HackerProfiles", "role", {
+      type: Sequelize.ENUM,
+      values: ["hacker", "admin", "sponsor", "volunteer"],
+      defaultValue: "hacker"
+    })
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.changeColumn("HackerProfiles", "role", {
+      type: Sequelize.ENUM,
+      values: ["hacker", "admin", "sponsor", "superadmin"],
+      defaultValue: "hacker"
+    })
+  }
+};
```

Comment:
@WilhelmWillie I'm not actually sure, I imagine updating a column resets the values? This could potentially cause data loss on prod

Reply:
Didn't reset tables when I migrated, so it should be fine. Maybe try to pull the repo and migrate on your dbs?

Reply:
i've done enum changes before and don't remember losing data. worst case scenario: if it does, just run a SQL script that sets everyone's role to hacker since rn we only have hackers and admins

Reply:
Just ran it against my own db, it doesn't change existing enums